schema/filter_artworks.js

```diff
@@ -132,13 +133,22 @@ function filterArtworks(primaryKey) {
       },
     },
     resolve: (root, options, request, { rootValue: { accessToken } }) => {
+      const gravityOptions = options;
       if (primaryKey) {
-        options[primaryKey] = root.id; // eslint-disable-line no-param-reassign
+        gravityOptions[primaryKey] = root.id;
       }
+
+      // Support queries that have an empty, or * medium which indicates
+      // that we want all mediums. This results in us removing the query option
+      // making the graphQL queries between all and a subset of mediums the same shape.
+      if (options.medium === '' || options.medium === '*') {
```

Comment:
Let’s settle on 1 sentinel, which I think should be `*` as to be consistent with the prime range sentinels.